examples/prometheus-operator/prometheus.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  labels:
+    prometheus: basic
+  name: basic
+  namespace: monitoring
+spec:
+  alerting:
+    alertmanagers:
+    - name: alertmanager-main
+      namespace: monitoring
+      port: web
+  thanos:
```

Comment:
I don't think we should add thanos in this example. Since you don't set up other things like thanos query, obj store, etc.

Reply:
More companies use thanos , so this is neccessary. And this's  just a example.

Reply:
The purpose of this example is to tell the user can integrate tidb monitoring with the existing monitoring system.

Reply:
The demo is just a guide to integrate tidbmonitor with Prometheus operator, right? So it is unnecessary to have thanos, this is just part of feature that prom operator has, but not the core feature we need in this example. If you really want to mention thanos, you can add a link to the thanos example in the prometheus-operator repo

Reply:
This demo is a guide to teach user integrate tidbmonitor with existing monitor system，which maybe contain thanos framework.

Reply:
I agreed with @yeya24 , thanos here is not necessary. This guide is to show how to monitor tidbcluster with prometheus-operator, so I think it should provide the same ability as tidbmonitor.